ADOPTERS.md

```diff
@@ -1,9 +1,24 @@
 # Pinniped Adopters
 
-These organizations are using Pinniped.
+If you're using Pinniped and want to add your organization to this
+list, [follow these directions](#adding-your-organization-to-the-list-of-adopters)!
 
-* [VMware Tanzu](https://tanzu.vmware.com/) ([Tanzu Mission Control](https://tanzu.vmware.com/mission-control))
+## Organizations using Pinniped
 
-If you are using Pinniped and are not on this list, you can open a [pull
-request](https://github.com/vmware-tanzu/pinniped/issues/new?template=feature-proposal.md)
-to add yourself.
+<a href="https://tanzu.vmware.com/tanzu" border="0" target="_blank"><img alt="vmware-tanzu" src="site/themes/pinniped/static/img/vmware-tanzu.svg" height="50"></a>
+
+<a href="https://kubeapps.com/" border="0" target="_blank"><img alt="kubeapps" src="site/themes/pinniped/static/img/kubeapps.svg" height="50"></a>
+
+## Solutions built with Pinniped
```

Comment:
Should we include a bullet for Tanzu Mission Control here like we do with Kubeapps?

Reply:
Oh, also should we include a bullet for Tanzu Kubernetes grid as well since the 1.3 release includes Pinniped? https://docs.vmware.com/en/VMware-Tanzu-Kubernetes-Grid/index.html

Reply:
Yup, I think we should. Will you send over 1-2 sentences for TMC and TKG about how Pinniped is used? Please n thank ya.

Reply:
Okey dokey what do ya think about this:

TMC uses Pinniped to provide a uniform authentication experience across all attached clusters.

TKG uses Pinniped to provide a seamless SSO experience across management and workload clusters.

Reply:
@ankeesler ready to be merged, I think. hah (hoping I did this all correctly)

Reply:
LGTM! Responded over Slack: https://kubernetes.slack.com/archives/C01BW364RJA/p1617029847303100?thread_ts=1616707240.281900&cid=C01BW364RJA